dora/src/masm/x64.rs

```diff
@@ -1082,6 +1123,13 @@ pub struct ForwardJump {
 mod tests {
     use super::*;
 
+    #[test]
+    fn test_float_cmp_total() {
```

Comment:
I don't think it is necessary to test the assembly-output here. IMHO your tests in tests/srt.dora are good enough. Otherwise we would have here a long sequence of machine code bytes, which is incredibly hard to update/read/maintain.